### PR TITLE
Added support for Resonate.is

### LINKED ISSUE #1558: Support Resonate.is
### Expected behavior

Support scrobbling for the Resonate streaming service.

Sample URLs
   https://beta.resonate.is/  (Unified player)
   https://resonate.is/album/138/ugress-it_came_from_beyond_eternal_september/

I will happily fund a Resonate account for any developer that wants to take this on.

### PR DESCRIPTION
**Pull request by inverse**
Resolves #1558

Should we add a test? It's a private site so how will that be handled?

### CODE REVIEW

**Comment by inverse** [comment]
Not sure why the build is failing - I didnt touch that file :(

File: src/connectors/resonate.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	Connector.playerSelector = '#player';
+
+	Connector.artistSelector = '#player .metas a';
+
+	Connector.trackSelector = '#player .name';
+
+	Connector.durationSelector = '.playlist-item--playing .duration';
+
+	Connector.getTrackArt = () => {
+		return $('.player-artwork > div').css('background-image').replace('url(', '').replace(')', '').replace(/"/gi, '');
```
Comment (alexesprit) [inline src/connectors/resonate.js]: This code could probably replaced by `Util.extractUrlFromCssProperty` function.

File: src/connectors/resonate.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	};
+
+	Connector.trackArtSelector = '.current-cover';
```
Comment (alexesprit) [inline src/connectors/resonate.js]: This property and another one below won't be used because of overriden `Connector.getTrackArt` function.

Comment (inverse) [reply src/connectors/resonate.js]: good catch!

**Comment by lindner** [comment]
Looks like beta3 has been promoted to beta.  (That was fast...)

@peteratomic - can you confirm that this is going to stay this way?

**Comment by lindner** [comment]
Also it would be nice if the Album Title was available in the DOM for use.

**Comment by peteratomic** [comment]
Looks like Augustin is a few days ahead of schedule! Yes I think the address will stay the same until we go out of beta status.

**Comment by inverse** [comment]
So shall I remove the beta3 routing and make this the normal flow? 

- Having album title in the dOM would a be a nice thing
- Also having the current duration would make scrobbling nicer, although not required

**Comment by lindner** [comment]
Hi folks, the resonate devs have added the album data into the DOM.  Here's an example:

<div class="flex items-end w-100">
<span class="track-name truncate b pv1">Lung Breathers</span>
<span class="track-album truncate pv1 pl1 f7">Coolboy</span></div>

**Comment by inverse** [comment]
@lindner  Updated

**Comment by lindner** [comment]
Changes LGTM

**Comment by lindner** [comment]
LGTM == Looks Good to Me.

**Pull request merged**